client/core/trade.go

```diff
@@ -1513,6 +1515,8 @@ func (t *trackedTrade) isRedeemable(ctx context.Context, match *matchTracker) (r
 				" be network latency.", match)
 		}
 		match.setSwapConfirms(int64(confs))
+		cord := t.coreOrder()
```

Comment:
I think these are going to deadlock.  `coreOrderInternal`?

Also can skip the `cord` variable.

And could use a few words in a trailing comment about the intent.